Phaethon/WebClient/Controllers/LoginController.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.Web;
+using System.Web.Mvc;
+using Core.Model;
+using Newtonsoft.Json;
+
+namespace WebClient.Controllers
+{
+    public class LoginController : Controller
+    {
+        private readonly HttpClient _client;
+
+        public LoginController()
+        {
+            _client = new HttpClient();
+            _client.BaseAddress = new Uri("http://localhost:64007/Login/");
+            
+        }
+
+        [HttpGet]
+        public ActionResult Index()
+        {
+            Session["Username"] = "";
+            return View();
+
+        }
+
+        [HttpPost]
+        public async Task<ActionResult> Index(Login loginModel)
+        {
+            var response = await _client.PostAsJsonAsync("Login", loginModel);
+            if (HttpStatusCode.OK == response.StatusCode)
+            {
+                var deserializedResponse = JsonConvert.DeserializeObject<int>(await response.Content.ReadAsStringAsync());
+                if (deserializedResponse != 0)
+                {
+                    Session["ID"] = deserializedResponse;
+                    Session["Username"] = loginModel.Username + " ";
```

Comment:
What is the pourpuse of Session "Username"?

Reply:
Session "Username" gives you the currently logged in person which is displayed on the top header